ingestion/functions/parsing/cuba/sample_data.json

```diff
@@ -0,0 +1 @@
 "info": "Hoy 11 de marzo de 2020 el Laboratorio Nacional de Referencia del IPK, informa que tres de los cuatro turistas aislados desde el d\u00eda de ayer resultaron positivos al Nuevo Coronavirus SARS CoV-2. Procedentes de la regi\u00f3n italiana de Lombard\u00eda con sintomatolog\u00eda respiratoria, se encontraban hospedados en un hostal en la ciudad de Trinidad, provincia Sancti Sp\u00edritus, y que hab\u00edan arribado por el aeropuerto internacional \u201cJos\u00e9 Mart\u00ed\u201d de La Habana en d\u00edas recientes. De forma inmediata fueron ingresados en el Instituto de Medicina Tropical Pedro Kour\u00ed", "provincias_visitadas": [], "dpacode_provincias_visitadas": []}, {"id": "it-2", "pais": "it", "edad": 57, "sexo": "mujer", "arribo_a_cuba_foco": "2020/03/10", "consulta_medico": "2020/03/10", "municipio_detecci\u00f3n": "Trinidad", "provincia_detecci\u00f3n": "Sancti Sp\u00edritus", "dpacode_municipio_deteccion": "28.06", "dpacode_provincia_deteccion": "28", "contagio": "importado", "contacto_focal": null, "centro_aislamiento": "ipk", "centro_diagnostico": "ipk", "posible_procedencia_contagio": ["it"], "info": "Hoy 11 de marzo de 2020 el Laboratorio Nacional de Referencia del IPK, informa que tres de los cuatro turistas aislados desde el d\u00eda de ayer resultaron positivos al Nuevo Coronavirus SARS CoV-2. Procedentes de la regi\u00f3n italiana de Lombard\u00eda con sintomatolog\u00eda respiratoria, se encontraban hospedados en un hostal en la ciudad de Trinidad, provincia Sancti Sp\u00edritus, y que hab\u00edan arribado por el aeropuerto internacional \u201cJos\u00e9 Mart\u00ed\u201d de La Habana en d\u00edas recientes. De forma inmediata fueron ingresados en el Instituto de Medicina Tropical Pedro Kour\u00ed", "provincias_visitadas": [], "dpacode_provincias_visitadas": []}, {"id": "it-3", "pais": "it", "edad": 60, "sexo": "mujer", "arribo_a_cuba_foco": "2020/03/10", "consulta_medico": "2020/03/10", "municipio_detecci\u00f3n": "Trinidad", "provincia_detecci\u00f3n": "Sancti Sp\u00edritus", "dpacode_municipio_deteccion": "28.06", "dpacode_provincia_deteccion": "28", "contagio": "importado", "contacto_focal": 5, "centro_aislamiento": "ipk", "centro_diagnostico": "ipk", "posible_procedencia_contagio": ["it"], "info": "Hoy 11 de marzo de 2020 el Laboratorio Nacional de Referencia del IPK, informa que tres de los cuatro turistas aislados desde el d\u00eda de ayer resultaron positivos al Nuevo Coronavirus SARS CoV-2. Procedentes de la regi\u00f3n italiana de Lombard\u00eda con sintomatolog\u00eda respiratoria, se encontraban hospedados en un hostal en la ciudad de Trinidad, provincia Sancti Sp\u00edritus, y que hab\u00edan arribado por el aeropuerto internacional \u201cJos\u00e9 Mart\u00ed\u201d de La Habana en d\u00edas recientes. De forma inmediata fueron ingresados en el Instituto de Medicina Tropical Pedro Kour\u00ed", "provincias_visitadas": [], "dpacode_provincias_visitadas": []}], "sujetos_riesgo": 30, "asintomaticos_numero": 0}}}}
```

Comment:
can you pretty print this please? it will be easier to debug failing tests this way